xonsh/base_shell.py

```diff
@@ -30,6 +30,7 @@ def close(self):
 
     def write(self, data):
         """Writes data to the original stdout and the buffer."""
+        data = data.replace('\001', '').replace('\002', '')
```

Comment:
Out of curiosity, why this addition?  Might there be a better place for this?  Maybe not...